### ISSUE TITLE
invoking super has changed, update the error message.

### ISSUE THREAD

**Issue by voodoologic** [emberjs/ember.js#18790]
Related to [this stack overflow.](https://stackoverflow.com/questions/55596958/ember-init-failing-to-recognize-super).  I'm using 3.16 and I made a custom class.  The error I got was to, invoke super in my init method.  But as you can see from stack overflow, the way super was called has changed.  I want to fix the error and do a pull request, but I'm still a n00b at javascript.

**Comment by voodoologic** [comment]
https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet#component-properties shows the difference.

**Comment by tonyown1** [comment]
I'm assuming you're using Ember octane since you're using a custom class. Indeed, in the example you've shown, the way to call the init now is by using the 
`constructor(args) {
    super(...arguments);
}`

**Comment by voodoologic** [comment]
Correct.  I can't insert a screenshot, but I can assure you that the browser console shows the wrong remedy for octane (3.16).

**Comment by rwjblue** [comment]
Yeah, we should probably change the default error based on if we detect that the various octane flags are set.

**Comment by johanrd** [comment]
I think this can be closed, as this seems to be fixed by 60c4b6009d

**Comment by NullVoxPopuli** [comment]
yup -- tyty for checking in on this!
